source/features.html.erb

```diff
@@ -0,0 +1,174 @@
+  </li>
+</ol>
+
+<h2 class="govuk-heading-l">Features available now</h2>
+<p>
+  During this early phase of development, we are focusing on the simplest and
+  most commonly needed form features first.
+</p>
+
+<h3 class="govuk-heading-m">Create simple custom questions</h3>
+
+<p>You can create custom questions to ask for:</p>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li>a single line of text</li>
+  <li>a date</li>
+  <li>a UK address</li>
+  <li>a National Insurance number</li>
+  <li>a phone number</li>
+  <li>an email address</li>
+</ul>
+
+<p>
+  When people complete the form, their responses will be validated to make sure
+  they’re in the correct format. You can also add a hint to a question to help
+  people answer it.
+</p>
+
+<h3 class="govuk-heading-m">
+  Forms use GOV.UK Design System styles, components and patterns
+</h3>
+
+<p>
+  Using these tested and accessible designs means you do not have to worry about
+  the visual and interface design of your forms. And they’ll automatically look
+  consistent with GOV.UK.
+</p>
+
+<h3 class="govuk-heading-m">Provide privacy information</h3>
+
+<p>
+  To comply with the UK General Data Protection Regulation (UK GDPR), you’ll
+  need to provide a link to privacy information for the people who will enter
+  their data into your form.
+</p>
+
+<p>
+  The privacy information needs to include your purposes for collecting the
+  data, and how you will manage the data. You may already publish this
+  information, for example in a privacy notice or personal information charter.
+</p>
+
+<p>
+  A link to the privacy information will be added to the footer of your form.
+</p>
+
+<h3 class="govuk-heading-m">Set the email address for completed forms</h3>
+
+<p>
+  Each form needs an email address to be set for completed forms to be sent to
+  when they’re submitted. Data from completed forms will be sent to this email
+  address in the body of an email.
+</p>
+
+<h3 class="govuk-heading-m">
+  People can check their answers before they submit a form
+</h3>
+
+<p>
+  People will be shown a summary page when they have answered all the questions
+  in a form. They will be able to review and check their answers before they
+  submit the form. This will help to avoid errors or missing information.
+</p>
+
+<h3 class="govuk-heading-m">Tell people what will happen next</h3>
+
+<p>
+  Content you add about what will happen after the form is submitted will be
+  shown on a confirmation page when someone submits a form. This will help to
+  set their expectations and avoid unnecessary contact.
+</p>
+
+<h2 class="govuk-heading-l">Features we’re working on next</h2>
+
+<p>These are the features we’re working on now and next:</p>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li>
+    Adding a declaration for people to agree to when they submit their responses
+    - this can be used in place of a signature
+  </li>
+  <li>Providing contact details for people to get help</li>
```

Comment:
🤔  I think we can mark this as done?

Reply:
Good point. Will need some content work to make it match the other items in the section above